bot/codeSharingBot.ts

```diff
@@ -20,10 +22,37 @@ export class CodeSharingBot extends TeamsActivityHandler {
     // Unfurling link contains `github`.
     if (url.includes("github.com")) {
       return await handleGitHubUrl(url);
+    } else if (url.includes(".visualstudio.com")) {
+      const adapter = context.adapter as CloudAdapter;
+      const userTokenClient = context.turnState.get(adapter.UserTokenClientKey);
```

Comment:
How about using  `get<UserTokenClient>` to make the type for `userTokenClient` explicit.

Reply:
Resolved. Thanks~